### ISSUE TITLE
Front end vs Back end

### ISSUE THREAD

**Issue by garcialo** [jlem/WebDevChallenge#10]
What do you think about creating separate documents for front end vs. back end challenges?

I think it would make it easier for users interested in either to quickly filter out challenges they don't want to do.

It would also allow inclusion of global best-practices; for example, making sure front end code validates. http://w3c.github.io/developers/tools/#tools

**Comment by niekvandepas** [comment]
I agree with this; in addition, I feel like backend challenges should be language-independent.

**Comment by arnoldstoba** [comment]
Could surely be a nice idea to create a general separation between front- and backend. Some javascript tasks should surely be represented in both sections tho, possibly the same for PHP.

**Comment by AntMooreWebDev** [comment]
> I feel like backend challenges should be language-independent.
> - NiekPas

I second this.

I think that separating the tasks will help and particularly if additional information/help is going to provided with the tasks.

**Comment by jlem** [comment]
I agree that we'll need some way of organizing these challenges as the list grows. It's already hard to maintain a single file!

Is there a benefit to splitting it up even further, beyond front and backend?

**Comment by AntMooreHroc** [comment]
Perhaps it would be beneficial to breakdown the front-end into languages at some point, if the list becomes much longer?
